internal/pkg/cli/init.go

```diff
@@ -153,8 +171,8 @@ func (opts *InitOpts) loadApp() error {
 	return opts.initApp.Validate()
 }
 
-// deploy prompts the user to deploy a test environment if the project doesn't already have one.
-func (opts *InitOpts) deploy() error {
+// deployEnv prompts the user to deployEnv a test environment if the project doesn't already have one.
```

Comment:
Nit: second "deploy" shouldn't be changed to "deployEnv" right?

Reply:
Ah yes, the woes of replace all.